src/layer/S1GRDEOCloudLayer.ts

```diff
@@ -95,11 +98,16 @@ export class S1GRDEOCloudLayer extends AbstractSentinelHubV1OrV2Layer {
     );
   }
 
-  protected getFindTilesAdditionalParameters(): Record<string, string> {
-    return {
+  protected getFindTilesAdditionalParameters(): Record<string, any> {
+    const result = {
       productType: 'GRD',
       acquisitionMode: this.acquisitionMode,
       polarization: this.polarization,
+      orbitDirection: this.orbitDirection,
     };
+    if (this.orbitDirection !== null) {
+      result.orbitDirection = this.orbitDirection;
+    }
```

Comment:
One of those is unnecessary, either put the `orbitDirection` in the `result` without checking or put it when checking but not before.

Reply:
Thank you, fixed.

Reply:
Nice. Resolving.